Add membership (#1)
oam-dev/community: community-membership.md

# Community Membership [Draft]

*This doc is in pre-release, it will be offcially released after 3 months from the date of its first commit*

All contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.

This doc outlines the various responsibilities of contributor roles in

> KubeVela.

Reviewer: merge with the above line

| Role | Responsibilities | Requirements | Defined by |
| -----| ---------------- | ------------ | -------|

> | Member | Active contributor in the community | have made multiple contributions to the project | OAM GitHub org member|

Reviewer: ```suggestion
| Member | Active participation in the community | have made multiple contributions to the project | OAM GitHub org member|
```

> | Reviewer | Review contributions |  Active contributor and/or code reviewer. Responsible with major features | [OWNERS] file reviewer entry |

Reviewer: ```suggestion
| Reviewer | Review contributions |  Active contributor and/or code reviewer. Responsible for major features | [OWNERS] file reviewer entry |
```

> | Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor. On charge of certain product area| [OWNERS] file approver entry|

Reviewer: ```suggestion
| Approver | Approve and commit contributions | Highly experienced active reviewer and contributor. On charge of certain project domain(s) | [OWNERS] file approver entry|
```
